catalogue/webapp/components/ItemRequestModal/ItemRequestModal.tsx

```diff
@@ -11,7 +31,26 @@ import { classNames, font } from '@weco/common/utils/classnames';
 import LL from '@weco/common/views/components/styled/LL';
 import { allowedRequests } from '@weco/common/values/requests';
 import RequestingDayPicker from '../RequestingDayPicker/RequestingDayPicker';
-import { useToggles } from '@weco/common/server-data/Context';
+import { convertDayNumberToDay } from '../../utils/dates';
+
+function arrayofItemsToText(arr) {
+  if (arr.length === 1) return arr[0];
+  const initialItems = arr.slice(0, arr.length - 1);
+  const lastItem = arr[arr.length - 1];
```

Comment:
This reminds me that I heard about [at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) the other day. Might be something we want to start using